lib/mongodb.js

```diff
@@ -816,18 +843,83 @@ MongoDB.prototype.all = function all(model, filter, options, callback) {
     query = self.buildWhere(model, filter.where);
   }
   var fields = filter.fields;
+  var relations = this._models[model].settings.relations;
```

Comment:
@shalkam We don't contain/expose model or model instance in connector, so I believe there is no proper way to get model settings/relations here.

Instead, you can provide the config info you need in `options` when calling `User.find({your_filter}, {your_options}, cb)`, then when `find()` calls `MongoDB.prototype.all()` under the hood, you get those config info:
```js
MongoDB.prototype.all = function all(model, filter, options, callback) { 
  // get relation info you need from `options`
}
```
details see my comment in 

And I would suggest call 'User.find({your_filter}, {your_options}, cb)' in test case instead of calling `User.all()`, `all` is an alias of `find` but we usually use `find` in tests .